project-ideas_2023/README.md

```diff
@@ -0,0 +1,272 @@
+
+In [RADAR-pipeline](https://github.com/RADAR-base/radar-pipeline), we are working on an open-source feature extraction pipeline which is capable of processing and analysing raw data collected through smartphones and wearables. The applicable sensor modalities, as detailed by the RADAR-base platform schemas, include GPS, Bluetooth, phone and app usage, sleep, step count, heart rate and others.  We have shown through a [depression case study](https://kclpure.kcl.ac.uk/portal/en/publications/relationship-between-major-depression-symptom-severity-and-sleep-collected-using-a-wristband-wearable-device(e28ac77e-18f6-43b0-8f73-19da02024f23).html) the functionality and utility of the pipeline. The presented open-source library will reduce burdens for studies aiming to develop relevant digital biomarkers and encourage and facilitate reproducible research. It also provides a standardized framework for contributing new pipelines with an emphasis on reuse.
+
+![Example of features in radar-pipeline](https://www.ncbi.nlm.nih.gov/pmc/articles/instance/6902121/bin/nihms-1061609-f0001.jpg)
+
+**Current Status:**
+
+The framework is in development stage, and we are open to new and interesting ideas. We have a working module now and we in the process of finalizing the framework and make it production-ready.
+
+**Goals:**
+| Milestones                                         | Description                                                                                                                                             |
+|----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Extend the data reader library                     |<br>- Extend data reading from remote server <br>- Add funtionality to read files in various formats such as Avro, json etc |
+| Make feature processing more customizable          | Integrate parameter processing in features.                                                                                                             |
+| Add feature summarization and visualization module | <br>- Add summarization library, which provides summary statistics of features. <br>- Implement support for open source visualization library                       |
+| Develop module of reporting missing data           | Add a module that would report missing data.                                                                                                            |
+| Add industry-wide used features                    | Implement industry-wide features used for mobility, heart rate, phone-interaction etc                                                                   |
```

Comment:
@Hsankesara These are really good, but i think might be overwhelming for participants. Unless you plan to get extension for the work, i would reduce the goals or make some of them optional based on priority.